handbook/communication/index.md

```diff
@@ -41,6 +41,24 @@ The following places are not sources of truth. Treat documents and conversations
 - Always end on time (or early).
   - If there's more to discuss, handle it asynchronously or schedule a new meeting with the appropriate participants.
   - A meeting that would run over is a nudge that the meeting's agenda or duration was wrong, or that the meeting was led ineffectively.
+- When creating the Google Calendar event:
+  - Add a [Zoom video call](#video-calls).
+  - Enable "Guest permissions: Modify event" so that other people can easily reschedule the meeting if needed.
+- Add a new or ongoing Google Doc link to the event description for taking notes during the meeting.
```

Comment:
I think we should express a preference for putting notes in ongoing docs (where an appropriate one exists) to avoid the proliferation of many different docs. For example, I think I could make the argument that the notes for our meeting today could have gone in our regular team sync doc and didn't need its own doc.